component.html

```diff
@@ -134,6 +135,146 @@ <h3 class="avatar-heading">Alert</h3>
       </iframe>
     </section>
 
+
+    <!-- ---- Badges----  -->
+
+    <section class="main-title" id="badges">
+      <h2 class="avatar-heading">Badges</h2>
+      <p class="component-navbar-item">
+        Badges are used to highlight an item's status for quick recognition.Badges scale to match the size of the immediate parent element by using relative font sizing and rem units.
+      </p>
+      <h3 class="avatar-heading">Badge with Icons</h3>
+      <p class="component-navbar-item">
+        To show cart or notification count, you can use number badge. Check code below to copy the html part as is.
+      </p>
+      <div class="badge-items flex">
+        <button type="button" class="badge-button flex center badge-xl">
```

Comment:
why you used type=button?

Reply:
It came up because of auto complete, removed